prime-router/metadata/hl7_mapping/common/patient.yml

```
@@ -112,7 +112,7 @@ elements:
 
   # We have to check for system equals "email" because we're not receiving use for email
   - name: phone-number-home-mobile-email
-    resource: '%resource.telecom.where(use != "work" or system = "email")'
+    resource: '%resource.telecom.where(system = "email" or use.empty() or use != "work")'
```

Would this be fair? Not sure if we're seeing that `use == "work"` and `system == "email"` at the same time, but the comment implies that we shouldn't see that. If this is a safe change, we could also update the comment at L113 to say "assume 'home' if use is not set", to not reference email at all.
```suggestion
    resource: '%resource.telecom.where(use.empty() or use != "work")'
```

---

It is actually the only way ti works. It was previously causing a bug where when it wasn't equal to work but was missing it was dropping data. I can update the comment though.

---

Making this change doesn't drop data, but it does break one of the new XTN tests. But I think it's still a good change. I'll leave another comment on the breaking tests to demonstrate.